packages/uix-host-react/src/components/Extensible.tsx

```diff
@@ -77,33 +77,22 @@ export function Extensible({
   guestOptions,
   runtimeContainer,
   debug,
-  sharedContext = {},
```

Comment:
This little bit would cause cycling re-render if `sharedContext` is not specified, since this object is re-created on each render.

Reply:
We can assign this to a cached instance member that is an empty object, a `defaultSharedContext` perhaps.

Reply:
Yep, I could but host actually accepts `undefined` value and handles it correctly, so I think it's ok as is.